doc/help/elektra-spec.md

```diff
@@ -21,7 +21,7 @@ key database semantics (new plugins might be necessary, though).
 The most simple use is to enlist all keys that will be used by an application
 and maybe give a description for them (we use ini syntax in this document):
 
-```
+```ini
```

Comment:
Thanks for the info!

Is there a list which languages are supported? (like for github https://github.com/github/linguist/blob/master/lib/linguist/languages.yml)